docs/operator-manual/applicationset/Generators-Git.md

```diff
@@ -45,16 +45,19 @@ spec:
       - path: applicationset/examples/git-generator-directory/cluster-addons/*
   template:
     metadata:
-      name: '{{path[0]}}'
+      name: '{{path.basename}}'
     spec:
-      project: "my-project"
+      project: default
```

Comment:
Would you mind reverting this change? The use of `"my-project"` is meant to guide folks to think about which project they use instead of just relying on the default project. I think the default project gets overused, and it is overly-permissive for almost all use cases.

Reply:
Thank you for your review.
I agree with you.
I have reflected on your suggestions!